__fixtures__/example-todo-main/api/src/functions/healthz/healthz.js

```diff
@@ -0,0 +1,15 @@
+import { logger } from 'src/lib/logger'
+
+export const handler = async (event, context) => {
+  logger.info('Invoked hello function')
+
+  return {
+    statusCode: 200,
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      data: 'healtz function',
```

Comment:
```suggestion
      data: 'healthz function',
```

I mean, maybe "healtz" actually gets the point across even better 🤷‍♂️